packages/textfield/src/spectrum-textfield.css

```diff
@@ -22,15 +22,45 @@ THIS FILE IS MACHINE GENERATED. DO NOT EDIT */
         --spectrum-textfield-border-radius,
         var(--spectrum-alias-border-radius-regular)
     );
-    padding: 3px
-        var(
-            --spectrum-textfield-padding-x,
-            var(--spectrum-global-dimension-size-150)
+    padding: calc(
```

Comment:
Padding, font-family, and font-size all change slightly on textfield and derivatives (textarea, search, etc).